os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -111,10 +111,10 @@ bool osif_task_create(void **pp_handle, const char *p_name, void (*p_routine)(vo
 	task_info[2] = NULL;
 	stack_size = stack_size * sizeof(uint32_t);
 	priority = (priority + SCHED_PRIORITY_DEFAULT > SCHED_PRIORITY_MAX || priority + SCHED_PRIORITY_DEFAULT < SCHED_PRIORITY_MIN)? SCHED_PRIORITY_DEFAULT : priority + SCHED_PRIORITY_DEFAULT;

```

Comment:
`priority` is `uint16_t`, `SCHED_PRIORITY_DEFAULT` is `100` and `SCHED_PRIORITY_MIN` is `1`.
So `priority + SCHED_PRIORITY_DEFAULT` cannot be smaller than `SCHED_PRIORITY_MIN`.
Please check this.

Reply:
Done modify.